Add RTL support (closes #257 and closes #347)

ricardobrandao opened the pull request.

ricardobrandao commented:
Not sure what people thinks about inverting the dialog box position for RTL (i.e. right goes left and the opposite). If _everyone_ agrees I can also add those changes.

TimFerrell commented:
Cool work! Glad to see this finally getting implemented. Some feedback:

- There are no tests. I recognize that the JS footprint for this is very small, but please add some to check for the presence of the RTL setting. Bonus points for validating the CSS.
- Make text in the demo "Right-To-Left" instead of "Is Right-To-Left"
- I would also like to see an example of this using an RTL language. Can you please attach an image of this functionality using an RTL font?

ricardobrandao commented:
@TimFerrell thanks! Glad to help 👍 

I've fixed the demo and added some tests. 

Also, here it is the example:
<img width="408" alt="toastr-rtl" src="https://cloud.githubusercontent.com/assets/2292745/17006373/376fcb06-4ed8-11e6-8b06-cf7a2ba385a8.png">

TimFerrell commented on build/toastr.css:
```diff
@@ -178,24 +188,41 @@ button.toast-close-button {
   }
+  #toast-container .rtl .toast-close-button {
+    left: -0.2em;
+    right: 0.2em;
```
Should top?

ricardobrandao replied:
Already tops from the [non-rtl version](https://github.com/CodeSeven/toastr/pull/439/files#diff-cc044eb2261831f76b5653c793d7ecd8R196).

TimFerrell commented on build/toastr.css:
```diff
@@ -178,24 +188,41 @@ button.toast-close-button {
     top: -0.2em;
   }
+  #toast-container .rtl .toast-close-button {
+    left: -0.2em;
```
Should be top.

TimFerrell commented:
Thanks very much for the work!

ricardobrandao commented:
@TimFerrell np :)

Are there any chance of creating a release for this? We use jspm in our project and it would be great not to point directly to the **develop** branch.

TimFerrell commented:
Let me take a look over what changes we've got in develop and what's in the development pipeline. I think we've got enough in the diff to justify a new release to master, just want to confirm.
